.travis.yml

```diff
@@ -1,10 +1,8 @@
 language: node_js
 
 node_js:
-  - "0.10"
-  - "0.12"
-  - "4.2"
-  - "5.0"
+  - "4"
+  - "6"
```

Comment:
We want to track what eslint@3 supports, which is: https://github.com/eslint/eslint/blob/3.x/package.json#L119-L121

```js
  "engines": {
    "node": ">=4"
  }
```